spec/function.dd

```diff
@@ -670,28 +671,28 @@ $(H2 $(LNAME2 inout-functions, Inout Functions))
         ---
         )
 
-    $(P The $(D_KEYWORD inout) forms a wildcard that stands in for
-        any of mutable, const, immutable, inout, or inout const. When the
-        function is called, the inout of the return type is changed to whatever
-        the mutable, const, immutable, inout, or inout const status of the
-        argument type to the parameter inout was.
+    $(P The $(D_KEYWORD inout) token forms a wildcard that stands in for
+        any of mutable, `const`, `immutable`, `inout`, or `inout const`. When calling
+        the function, the `inout` of the return type is changed to whatever
+        the mutable, `const`, `immutable`, `inout`, or `inout const` status of the
+        argument type to the `inout` parameter was.
```

Comment:
While we're here, might as well fix the awkward wording of this sentence. At a minimum, removing "whatever" and the trailing "was" improves it.

A bigger issue is the use of "status". As worded, by listing all the possible states before the word "status", it looks like it's describing the status of each of *those* states, and not saying that one of those states will be applied. We'd be better off with this:

"the `inout` status of the return type is changed to that of the argument type to the `inout` parameter., one of [list all the states here]."

Reply:
Good point, I've rewritten it now.